tasks/main.yml

```diff
@@ -28,5 +28,13 @@
   notify:
     - Restart Zookeeper
 
+- name: Configure JAVA_OPTS
```

Comment:
Shoot, my bad, sorry for that. I think I may have left out some of the stuff in this commit. I'll get back to you once I review it, sorry again for the trouble!